packages/eslint-plugin/tests/rules/explicit-module-boundary-types.test.ts

```diff
@@ -416,7 +417,7 @@ export class Test {
       `,
       options: [
         {
-          allowedNames: ['prop', 'method', 'foo'],
+          allowedNames: ['prop', 'method', 'null', 'foo'],
```

Comment:
note that adding `'Symbol(a)'` to `allowedNames` allows both
`'Symbol(a)'() {}`
and
`[Symbol('a')]() {}`

Not sure if this is a feature or a bug, or if it's worth documenting? 🤷‍♂️

Reply:
This is no longer true with the latest change requested by @kirkwaiblinger.

Reply:
Since `Symbol('a')` is always a unique value, I don't think that it would make sense for the `allowedNames` to be able to specify it in this way.

Whether `Symbol.for('a')` should have a way of specifying it.... 🤷‍♂️. That could be sorted out in a feature request. In any case, I think it would belong in the rule logic and/or specifier logic to stringify the symbol, rather than in the static member access utility.